packages/core-data/src/selectors.js

```diff
@@ -178,5 +178,5 @@ export function isPreviewEmbedFallback( state, url ) {
  * @return {boolean} Upload Permissions.
  */
 export function hasUploadPermissions( state ) {
-	return state.hasUploadPermissions;
+	return !! state.hasUploadPermissions;
```

Comment:
What's the reason for this change? This should be something where, if we're expecting it to be a boolean, it should be forced to be a boolean in the reducer logic, not derived by the selector.

Reply:
Force-pushed this change. This is now handled at the reducer level where we default to `true` instead of `{}`. I chose `true` instead of `false`, since `{}` is a truthy value, and this should thus maintain existing behaviour.

Reply:
I'm guessing the previous implementation was a result of some bad copypasta from other reducers. A reducer value prefixed with "has" should almost certainly hold a boolean value.

Were there unit tests which should be updated as part of the change? Can there be, if not? 😄

Reply:
Nope, this particular reducer didn't seem to have any unit tests. I added a couple :)